src/System.IO.FileSystem/tests/FileStream/Read.cs

```diff
@@ -20,6 +20,12 @@ public void NullArrayThrows()
         }
 
         [Fact]
+        public void NegativeReadRootThrows()
+        {
+            Assert.Throws<UnauthorizedAccessException>(() => { new FileStream("\\", FileMode.Open, FileAccess.Read); });
```

Comment:
We'll see what the CI build says, but I'm guessing this test is going to fail on Linux, and will need to be attributed with [PlatformSpecific(PlatformID.Windows)].